### PR TITLE
Fix query params based omni api calls

### PR DESCRIPTION
**Pull request by ericanderson**

### CODE REVIEW

**Comment by ericanderson** [comment]
No changelog because we haven't shipped the omni change so its not needed.

File: packages/omniapi-generator/src/generateResource.ts [hunk 1]
```diff
@@ -132,12 +132,18 @@ function generateOperationArray(method: StaticOperation) {
   const requestType = method.requestBody?.body.requestType;
   const bodyInfo = getReqBodyInfo(requestType);
   const respInfo = getResponseInfo(method.response);
+  const paramInfo = getParamInfo(method.parameters);
+  paramInfo.QUERY.length;
```
Comment (ssanjay1) [inline packages/omniapi-generator/src/generateResource.ts:136]: Can we remove this line?

File: packages/omniapi/src/public/Datasets_Branch.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ const _listBranches: $OmniMethod<
     datasetRid: $C.DatasetRid,
     $queryParams?: { pageSize?: $C.PageSize; pageToken?: $C.PageToken },
   ) => Promise<$C.ListBranchesResponse>
-> = [0, "/v1/datasets/{0}/branches"];
+> = [0, "/v1/datasets/{0}/branches", 2];
```
Comment (ssanjay1) [inline packages/omniapi/src/public/Datasets_Branch.ts:29]: I'm confused as to why these numbers have been appended to the end in these generated files?

**Review by ssanjay1 — APPROVED** [review]

File: packages/omniapi/src/public/Datasets_Branch.ts (hunk 2 above)
Comment (ericanderson) [reply packages/omniapi/src/public/Datasets_Branch.ts:29]: They tell the runtime if there is query or header params. Its just bitwise math.

File: packages/omniapi-generator/src/generateResource.ts (hunk 1 above)
Comment (ericanderson) [reply packages/omniapi-generator/src/generateResource.ts:136]: OOps, yes thank you

**Pull request merged**